public/css/self-service-time-to-pay.css

```diff
@@ -513,4 +513,12 @@ details summary:focus,
 
 .no-details details summary .arrow-closed {
     display:none;
+}
+
+.font-xsmall-normal {
```

Comment:
Can you not use a hmrc library for this stuff?

Reply:
fixed